### PR TITLE
Webpage update for 7.0.18

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

File: docs/website/src/main/resources/README.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ sponsored by the Eclipse Foundation.
 
 ## Latest News
 
+
+## October 30, 2024 -- Eclipse GlassFish 7.0.18 Available
```
Comment (pzygielo) [inline docs/website/src/main/resources/README.md:13]: 30? :eyes:

Comment (OndroMih) [inline docs/website/src/main/resources/README.md:13]:
```suggestion
## October 2, 2024 -- Eclipse GlassFish 7.0.18 Available
```

File: docs/website/src/main/resources/README.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ sponsored by the Eclipse Foundation.
...
+
+The full GlassFish team is happy to present you another great GlassFish release; Eclipse GlassFish 7.0.18!
+
+Download links are available from the [GlassFish Download page](download.md).
+
+This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using java -jar glassfish-embedded.jar. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules nucleus-admin and glassfish, and fixed various issues that were reported by users.
```
Comment (OndroMih) [inline docs/website/src/main/resources/README.md:19]:
```suggestion
This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using `java -jar glassfish-embedded.jar`. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules `nucleus-admin` and `glassfish`, and fixed various issues that were reported by users.
```

File: docs/website/src/main/resources/download.md [hunk 3]
```diff
@@ -4,20 +4,20 @@
...
+GlassFish 7.0.18 is a final release, containing final Jakarta EE 10 APIs. It compiles and runs on JDK 11 to JDK 23. MicroProfile support requires JDK 17 or higher.
 
-In this summer 2024 release of GlassFish we did a lot of maintenance in which we will continue in future releases. We fixed several security issues and the build now passes also in Java 23. We welcome feedback - if you notice that something doesn't work as expected, feel free to create an issue or even better create your own pull request.
+This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using java -jar glassfish-embedded.jar. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules nucleus-admin and glassfish, and fixed various issues that were reported by users.
```
Comment (OndroMih) [inline docs/website/src/main/resources/download.md:9]:
```suggestion
This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using `java -jar glassfish-embedded.jar`. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules `nucleus-admin` and `glassfish`, and fixed various issues that were reported by users.
```

File: docs/website/src/main/resources/download_gf7.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
 # Eclipse GlassFish 7.x Downloads
 
 
+## GlassFish 7.0.18
+
+Eclipse GlassFish is an application server, implementing Jakarta EE. This release is corresponding with the Jakarta EE 10 specification, which is a major new feature release. Jakarta EE 10 requires JDK 11 as a minimum, but also works on JDK 17 and JDK 21.
+
+GlassFish 7.0.18 is a final release, containing final Jakarta EE 10 APIs. It compiles and runs on JDK 11 to JDK 23. MicroProfile support requires JDK 17 or higher.
+
+This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using java -jar glassfish-embedded.jar. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules nucleus-admin and glassfish, and fixed various issues that were reported by users.
```
Comment (OndroMih) [inline docs/website/src/main/resources/download_gf7.md:10]:
```suggestion
This release is all about an exciting new feature that allows running GlassFish Embedded from the command line using `java -jar glassfish-embedded.jar`. It supports configuration via command line arguments and/or config files in the current directory, such as setting the HTTP port, deploying applications passed as arguments and running arbitrary AsAdmin commands at boot. Aside of that we did a large amount of maintenance for the internal modules `nucleus-admin` and `glassfish`, and fixed various issues that were reported by users.
```

**Review by OndroMih — CHANGES_REQUESTED** [review]

**Review by OndroMih — APPROVED** [review]

File: docs/website/src/main/resources/README.md (hunk 1 above)
Comment (OndroMih) [reply docs/website/src/main/resources/README.md]: I fixed it.

**Review by avpinchuk — APPROVED** [review]

**Pull request merged**